main/survey/survey_question.php

```diff
@@ -434,9 +434,6 @@ public function preSave($formData)
 
         // Adding an answer
         if (isset($_POST['buttons']) && isset($_POST['buttons']['add_answer'])) {
-            if (isset($_REQUEST['type']) && 'multiplechoiceother' === $_REQUEST['type']) {
-                $counter--;
-            }
```

Comment:
For this multiplechoiceother type, it is decremented due to the "other" option that is not considered to be displayed in the form.
You can test it by editing a question of this type.